### ISSUE TITLE
Adding fields to docTypes

### ISSUE THREAD

**Issue by craigs100** [Shazwazza/Articulate#158]
Hi, 
Trying to add fields to Articulate docTypes and read them in the Master.cshtml. Failing miserably. The site doesn't have ModelsBuilder switched on but I switched it on to see if it made a difference but it didn't.

Any advice would be appreciated (Have tried Our and Slack already: https://our.umbraco.org/projects/starter-kits/articulate/discussions/79270-using-own-master-with-new-fields-added-to-articulate-doctypes) - might be holiday season!

I've decided not to try to use the site's master file but to modify an existing working theme. However, I need the editors to be able to chose the background image and a few other things to be consistent with the rest of the site.

Cheers,

Craig

**Comment by craigs100** [comment]
Found the answer here: https://www.marceldigital.com/blog/2015/09/how-to-setup-an-articulate-blog-package-to-an-existing-umbraco-layout. So this could be closed, though I'd suggest a "Site integration" section be added to the wiki to cover this common scenario ;)

**Comment by Shazwazza** [comment]
Yup, it's a very common question on Our, just need to invent some more time in my life to write more docs :)
